client/state/plans/test/selectors.js

```diff
@@ -51,94 +51,21 @@ describe( 'selectors', () => {
 		} );
 	} );
 
-	describe( '#getPlanPriceObject()', () => {
-		it( 'should return price plan object', () => {
+	describe( '#getPlanRawPrice()', () => {
+		it( 'should return annual raw price', () => {
 			const state = getStateInstance();
-			const priceObject = getPlanPriceObject( state, 1003 );
-			expect( priceObject ).to.eql( {
-				currencySymbol: '$',
-				decimalMark: '.',
-				dollars: 99,
-				cents: 0
-			} );
+			const price = getPlanRawPrice( state, 1003 );
+			expect( price ).to.eql( 99 );
```

Comment:
Is this test a catch to make sure we update the prices in the Calypso code? Wouldn't want to encourage people to "fix" the test by just updating the expected price and not the actual code.